recipes/stacks/2.3b/conda_build_config.yaml

```diff
@@ -0,0 +1,6 @@
+c_compiler:
+  - gcc      # [linux]
```

Comment:
This file can be removed, we did the migration.

Reply:
thanks